addons/common/fnc_currentUnit.sqf

```diff
@@ -0,0 +1,20 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_currentUnit
+
+Description:
+    Returns the currently controlled unit.
+    Different from "player" when remote controlling units via zeus.
```

Comment:
Either shorten to one sentence or separate with empty line for possible future improvements to documentation generation.